src/provider/Provider.ts

```diff
@@ -8,17 +8,37 @@ import { Account } from '../account/Account';
 import { AvailableCommands } from '..';
 import { IAccount } from '../account/IAccount';
 
+/**
+ * Provider class
+ *
```

Comment:
Could be cool to have a little explaination of what is the provider Class

Reply:
is this description ok for you ?